mui/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## v4.0.0 (unreleased)
+
+### Updated
+
+-   Removed dependency on @mui/styles. [#232](https://github.com/brightlayer-ui/icons/issues/232)
+-   `SvgEvPlugIec62196T3a` icon name changed to `SvgEvPlugIec62196T3A`
```

Comment:
The two renames should go under a 'Changed' sections, along with a mention of the new minimum required version of mui/material (whenever the path to that utility function was made)